packages/share/lib/share.dart

```diff
@@ -45,13 +45,14 @@ class Share {
     return channel.invokeMethod('share', params);
   }
 
-  static Future<void> shareFile(Uri uri, String mimeType) {
-    assert(uri != null);
-    assert(new File(uri.toFilePath()).existsSync());
+  static Future<void> shareFile(String filepath, String mimeType) {
```

Comment:
I'm thinking just a string filepath is better than the equivalent file or URI, since we can assert validity and generate the URI easily.

Reply:
OK, please name it just `path` as in https://api.dartlang.org/dev/2.0.0-dev.55.0/dart-io/File-class.html